storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -0,0 +1,30 @@
+[id="persistent-storage-csi-ebs"]
+= AWS Elastic Block Store CSI Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ebs
+
+toc::[]
+
+== Overview
+
+Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plug-ins using a standard interface without ever having to change the core Kubernetes code.
```

Comment:
nit: just FYI, upstream documentation use `plugin` instead of `plug-ins`: 

https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/

I don't know which one is more correct :)

Reply:
> nit: just FYI, upstream documentation use `plugin` instead of `plug-ins`:
> 
> https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/
> 
> I don't know which one is more correct :)

I do prefer `plugin` myself, but we conform to the IBM Style Guide, which uses `plug-in`. :)